### DISCUSSION TITLE
Black outline problem

### DISCUSSION THREAD

**Discussion by ALMAZ00** [vmware/clarity#6717] [General]
Hello! Please help with black outline issue on some Clarity Design components.  The following are examples of components that exhibit this problem.
![image](https://user-images.githubusercontent.com/58396085/154928473-d8b40803-a132-4593-81c9-42335ec58fa8.png)
![image](https://user-images.githubusercontent.com/58396085/154928545-0299dec5-d18d-4191-967c-757cf904d2b7.png)
The black outline appears when clicking on the "three dots" and other components.
Please tell me how to remove this outline from all components of the Clarity Design System. Thanks in advance for your reply!

**Comment by Jinnie** [comment]
In which browser is this observed?
It's not something we've seen already.
We can hardly diagnose it without Stackblitz reproduction, please use one of our templates here as a starter:
https://stackblitz.com/@clr-team
